tests/test_kernels/test_kernels.py

```diff
@@ -25,11 +25,7 @@ def data(random):
 def test_constant(data):
     x1, x2 = data
 
-    # Check for dimension issues, either when instantiated...
-    with pytest.raises(ValueError):
-        kernels.Constant(jnp.ones(3))
```

Comment:
Instead of removing this, could we instead check that it raises when the kernel's `evaluate` method is called?